src/hooks/useApolloQuery.ts

```diff
@@ -69,7 +73,6 @@ export function useHolaplexInfiniteScrollQuery<
   const onCompleted: (raw: TGraphQLDataObject) => void = useCallback(
     (raw) => {
       const rawDataArray: TGraphQLElement[] | undefined = listExtractor(raw);
-      setHasMore(rawDataArray != null && rawDataArray.length > 0);
```

Comment:
This was the previous location looking to handle the has more but once there is data returned this is always greater than 0.